packages/strapi-plugin-i18n/controllers/locales.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+const { setCreatorFields, sanitizeEntity } = require('strapi-utils');
+const { getService } = require('../utils');
+const { validateCreateLocaleInput, validateUpdateLocaleInput } = require('../validation/locales');
+const { formatLocale } = require('../domain/locale');
+
+const listLocales = async ctx => {
```

Comment:
I would create a sanitizeLocale fn so we don't need to getModel et call sanitizeEntity in each handler